deployment.md

```diff
@@ -3,6 +3,21 @@
 * Continuous integration with VSO.
 * Docker and containers.
 * Cross-platform remote debugging.
+
+### Keeping Applications Running
+* [PM2](http://pm2.keymetrics.io/)
+* [forever](https://www.npmjs.com/package/forever)
+* [node-cluster-service](https://github.com/godaddy/node-cluster-service)
+
```

Comment:
Any chance you could add some short descriptions here? Or at least one catch all description at the top describing the value add / scenarios in which you'd use something like this?